packages/gutenpride-template/src/src/edit.js.mustache

```diff
@@ -0,0 +1,29 @@
+/**
+ * Retrieves the translation of text.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-i18n/
+ */
+import { Placeholder, TextControl } from '@wordpress/components';
```

Comment:
The comment is related to `@wordpress/i18n`. There are 2 options:
- we remove it
- we add another comment for `@wordpress/components`

It might be fine to use less annotations in this template since it has a corresponding tutorial.

Reply:
I added a comment.